app/src/main/java/com/keylesspalace/tusky/receiver/SendStatusBroadcastReceiver.kt

```diff
@@ -45,22 +45,19 @@ class SendStatusBroadcastReceiver : BroadcastReceiver() {
     override fun onReceive(context: Context, intent: Intent) {
         AndroidInjection.inject(this, context)
 
-        val notificationId = intent.getIntExtra(NotificationHelper.KEY_NOTIFICATION_ID, -1)
-        val senderId = intent.getLongExtra(NotificationHelper.KEY_SENDER_ACCOUNT_ID, -1)
-        val senderIdentifier = intent.getStringExtra(NotificationHelper.KEY_SENDER_ACCOUNT_IDENTIFIER)
-        val senderFullName = intent.getStringExtra(NotificationHelper.KEY_SENDER_ACCOUNT_FULL_NAME)
-        val citedStatusId = intent.getStringExtra(NotificationHelper.KEY_CITED_STATUS_ID)
-        val visibility = intent.getSerializableExtra(NotificationHelper.KEY_VISIBILITY) as Status.Visibility
-        val spoiler = intent.getStringExtra(NotificationHelper.KEY_SPOILER) ?: ""
-        val mentions = intent.getStringArrayExtra(NotificationHelper.KEY_MENTIONS) ?: emptyArray()
-        val citedText = intent.getStringExtra(NotificationHelper.KEY_CITED_TEXT)
-        val localAuthorId = intent.getStringExtra(NotificationHelper.KEY_CITED_AUTHOR_LOCAL)
-
-        val account = accountManager.getAccountById(senderId)
+        if (intent.action == NotificationHelper.REPLY_ACTION) {
```

Comment:
I'm a bit confused, why do we compose directly but reply via `BroadcastReceiver` still?

Reply:
Ah I forgot about direct reply, I am kinda stuck in Android 10 for now...